Sources/KlaviyoSwift/InternalAPIModels.swift

```diff
@@ -391,7 +391,7 @@ struct LegacyEvent: Equatable {
 
         if eventName == "$opened_push" {
             // Special handling for $opened_push include push token at the time of open
-            eventProperties["push_token"] = state.pushToken
+            eventProperties["push_token"] = state.pushTokenData?.pushBackground
```

Comment:
Whats this do?

Reply:
that looks like a bug 🤦🏻

Reply:
seems like this would result in `push_token: "ENABLED"`

Reply:
Oh maybe you are commenting on an old commit?

Reply:
I don't think so... it says `eventProperties["push_token"] = state.pushTokenData?.pushBackground`

Reply:
I changed how the state was arranged a bit so it could be diffed against token/device updates.

Reply:
should be `eventProperties["push_token"] = state.pushTokenData?.pushToken` ?

Reply:
Nvm...I thought when I clicked on the link it would take me to the branch but I was looking at master. It's wrong I will fix it!

Reply:
That coulda been caught by a unit test, maybe we're missing some detail in push opened event coverage